src-tauri/src/northstar/mod.rs

```diff
@@ -122,3 +125,56 @@ pub fn launch_northstar(
         get_host_os()
     ))
 }
+
+/// Prepare Northstar and Launch through Steam using the Browser Protocol
+#[tauri::command]
+pub fn launch_northstar_steam(
+    game_install: GameInstall,
+    _bypass_checks: Option<bool>,
+) -> Result<String, String> {
+    if !matches!(game_install.install_type, InstallType::STEAM) {
+        return Err("Titanfall2 was not installed via Steam".to_string());
+    }
+
+    match steamlocate::SteamDir::locate() {
+        Some(mut steamdir) => {
+            if get_host_os() != "windows" {
+                let titanfall2_steamid: u32 = TITANFALL2_STEAM_ID.parse().unwrap();
+                match steamdir.compat_tool(&titanfall2_steamid) {
+                    Some(compat) => {
+                        if !compat
+                            .name
+                            .clone()
+                            .unwrap()
+                            .to_ascii_lowercase()
+                            .contains("northstarproton")
```

Comment:
I assume that check will be expanded later with Proton-GE?
(can be done in separate PR)

Reply:
Depends on how things will be going forward.
I recall the plan being to modify the GE-Proton 8-4 release, add the wsock override and some more.

Idk if Tony is intended to rebrand it as NorthstarProton or not, we'll have to see.

Reply:
Aight, for now we'll just keep it as is I guess ^^